tests/SeleniumTests/__init__.py

```diff
@@ -11,6 +12,7 @@
 SELENIUM_GRID_HOST = os.environ.get('SELENIUM_GRID_HOST', 'localhost')
 SELENIUM_GRID_PORT = os.environ.get('SELENIUM_GRID_PORT', '4444')
 WEB_DRIVER_WAIT_TIMEOUT = int(os.environ.get('WEB_DRIVER_WAIT_TIMEOUT', 60))
+SE_NODE_SESSION_TIMEOUT = int(os.environ.get('SE_NODE_SESSION_TIMEOUT', 10))
```

Comment:
We do not need to test this here. It is already being tested in the upstream project. The issue was that we did not pass the value to the Node.